src/App.css

```diff
@@ -805,6 +805,20 @@ footer .developed-by {
   margin-bottom: 20px;
 }
 
+.quoteStyling {
+  font-family: 'Lato' , sans-serif;
+  text-align: center;
+  display: block;
+  color: black;
+  font-size: 32px;
+  border: 0;
```

Comment:
A couple of these things probably aren't necessary. For example, 'border: 0;' or 'color: black;' shouldn't be having any affect